src/ios/PushPlugin.m

```diff
@@ -1,16 +1,16 @@
 /*
  Copyright 2009-2011 Urban Airship Inc. All rights reserved.
-
+ 
```

Comment:
Can you fix all of the formatting problems with the file? It will make the review a lot easier.